src/js/resources.js

```diff
@@ -1,9 +1,17 @@
 // Hard-coded URLs to resources of interest.
 // TODO: this could be driven by a separate domain.
+
 module.exports = {
-    etl: origins.endpoint + '/domains/pedsnet.dcc.loader/facts',
-    warehouse: origins.endpoint + '/domains/pedsnet.dcc.warehouse/facts',
-    pcornet: origins.endpoint + '/domains/pcornet.cdm/facts',
-    pedsnet: origins.endpoint + '/domains/pedsnet.cdm/facts',
-    i2b2: origins.endpoint + '/domains/i2b2.data/facts'
-};
+    models: ['pedsnet',
+             'pcornet',
+             'i2b2',
+    ],
+
+    urls: {
```

Comment:
This URLs should be configurable which means they need to be defined at runtime when the host HTML file loads. That was the purpose of [this block](https://github.com/PEDSnet/pedsnet-catalogue/pull/1/files#diff-d4108beafd71e30c5be9d9a05026f01dL49) since it setups the configuration prior to loading the scripts. This module can reference the configuration and clean and copy the urls for other modules to use.